modular_skyrat/modules/colony_fabricator/code/clothing/storage.dm

```diff
@@ -0,0 +1,88 @@
+// Backpacks
+
+/obj/item/storage/backpack/industrial/frontier_colonist
+	name = "frontier backpack"
+	desc = "A rugged backpack often used by settlers and explorers. Holds all of your equipment and then some."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing.dmi'
+	icon_state = "backpack"
+	worn_icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	worn_icon_teshari = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn_teshari.dmi'
+	worn_icon_state = "backpack"
+	inhand_icon_state = null
+	// These have the storage of a duffelbag, however do not have the benefits that zipping/unzipping give a duffelbag (like bulky item storage)
+	storage_type = /datum/storage/duffel
```

Comment:
So it's basically just a flatly better backpack? I don't really see how that's balanced, or why everyone wouldn't want to use this. This feels like an easy choice for anyone that wants more storage, over just wearing a duffel bag, as those are still more inconvenient than this.

Reply:
Not sure why I did that one, its just normal backpack storage now.